### PR TITLE
fix (SQLite): Update SQLite dependencies - Resolve Exception in #577

### LINKED ISSUE #577: Exception during PerformUpgrade using LogScriptOutput with SQLite
I am getting this issue in .Net 5 and .Net Core 3.1, but not .Net Framework 4.7.2.

Running a simple test program:

```
            var connectionString = @"Data Source=:memory:";
            var result = DeployChanges.To
                .SQLiteDatabase(connectionString)
                .WithScriptsEmbeddedInAssembly(Assembly.GetEntryAssembly())
                .LogToConsole()
                .LogScriptOutput()
                .Build()
                .PerformUpgrade();
```

And using just this SQL script:

```
create table test (
    contact_id INTEGER PRIMARY KEY
);
```
Produces this output:

```
Beginning database upgrade
Checking whether journal table exists..
Journal table does not exist
Executing Database Server script 'dbup.SchemaMigrations.0000001_Initial.sql'
Checking whether journal table exists..
Creating the [SchemaVersions] table
The [SchemaVersions] table has been created
Exception has occured in script: 'dbup.SchemaMigrations.0000001_Initial.sql'
System.ArgumentNullException: SafeHandle cannot be null. (Parameter 'pHandle')
   at System.StubHelpers.StubHelpers.SafeHandleAddRef(SafeHandle pHandle, Boolean& success)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.Sqlite3_sqlite3.sqlite3_column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.Sqlite3_sqlite3.column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.sqlite3_column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.SqliteDataReader.get_FieldCount()
   at DbUp.Support.ScriptExecutor.WriteReaderToLog(IDataReader reader)
   at DbUp.Support.ScriptExecutor.ExecuteAndLogOutput(IDbCommand command)
   at DbUp.Support.ScriptExecutor.<>c__DisplayClass18_2.<Execute>b__1()
   at DbUp.SQLite.SQLiteScriptExecutor.ExecuteCommandsWithinExceptionHandler(Int32 index, SqlScript script, Action executeCommand)
   at DbUp.Support.ScriptExecutor.<>c__DisplayClass18_0.<Execute>b__0(Func`1 dbCommandFactory)
   at DbUp.Engine.Transactions.NoTransactionStrategy.Execute(Action`1 action)
   at DbUp.Engine.Transactions.DatabaseConnectionManager.ExecuteCommandsWithManagedConnection(Action`1 action)
   at DbUp.Support.ScriptExecutor.Execute(SqlScript script, IDictionary`2 variables)
Upgrade failed due to an unexpected exception:
System.ArgumentNullException: SafeHandle cannot be null. (Parameter 'pHandle')
   at System.StubHelpers.StubHelpers.SafeHandleAddRef(SafeHandle pHandle, Boolean& success)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.Sqlite3_sqlite3.sqlite3_column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.Sqlite3_sqlite3.column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.Interop.NativeMethods.sqlite3_column_count(Sqlite3StmtHandle stmt)
   at Microsoft.Data.Sqlite.SqliteDataReader.get_FieldCount()
   at DbUp.Support.ScriptExecutor.WriteReaderToLog(IDataReader reader)
   at DbUp.Support.ScriptExecutor.ExecuteAndLogOutput(IDbCommand command)
   at DbUp.Support.ScriptExecutor.<>c__DisplayClass18_2.<Execute>b__1()
   at DbUp.SQLite.SQLiteScriptExecutor.ExecuteCommandsWithinExceptionHandler(Int32 index, SqlScript script, Action executeCommand)
   at DbUp.Support.ScriptExecutor.<>c__DisplayClass18_0.<Execute>b__0(Func`1 dbCommandFactory)
   at DbUp.Engine.Transactions.NoTransactionStrategy.Execute(Action`1 action)
   at DbUp.Engine.Transactions.DatabaseConnectionManager.ExecuteCommandsWithManagedConnection(Action`1 action)
   at DbUp.Support.ScriptExecutor.Execute(SqlScript script, IDictionary`2 variables)
   at DbUp.Engine.UpgradeEngine.PerformUpgrade()
```

If I remove the `LogScriptOutput()` call the command works, so I can do that, but without it I lose my upgrade debug logging.

I'm using the latest versions of .Net 5 and 3.1 SDKs and `dbup-sqlite` v4.5.0 from NuGet.

### PR DESCRIPTION
**Pull request by jafin**
Update SQLite dependencies - Resolve Exception during PerformUpgrade using LogScriptOutput with SQLite Resolves DbUp/dbup-sqlite#2

- Bump target from netstandard 1.3;net462 to netstandard 2.0;net462
- Use System.Data.SQLite.Core 1.0.118
- Drop conditional preprocessors for MONO/NETCORE
- Update Sample, remove explicit create, should occur by default when init connection.
- Drop sqlite.mono project relates to DbUp/DbUp#655 supported frameworks.

## Breaking changes
- Dropped sqlite.mono
- Drop netstandard 1.3

### CODE REVIEW

File: src/dbup-sqlite/dbup-sqlite.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Description>DbUp makes it easy to deploy and upgrade SQL Server databases. This package adds SQLite support.</Description>
     <Title>DbUp SQLite Support</Title>
     <Authors>Paul Stovell;Jim Burger;Jake Ginnivan;Damian Maclennan</Authors>
-    <TargetFrameworks>netstandard1.3;net40;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```
Comment (sungam3r) [inline src/dbup-sqlite/dbup-sqlite.csproj:7]:
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```

File: src/dbup-tests-sqlite/dbup-tests-sqlite.csproj [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RootNamespace>DbUp.Tests.SQLite</RootNamespace>
+
+    <IsPackable>false</IsPackable>
+
```
Comment (sungam3r) [inline src/dbup-tests-sqlite/dbup-tests-sqlite.csproj:8]:
```suggestion
```

**Comment by sungam3r** [comment]
@jafin  Please rebase onto `master` if you are still interested.

**Comment by jafin** [comment]
@sungam3r  sure rebased.  See notes above re sqlite-mono. (Appears sqlite is only package still targeting mono, everything else looks like it has dropped it)

**Comment by jafin** [comment]
See https://github.com/DbUp/dbup-sqlite/pull/5

**Pull request closed without merging**